Std/Lean/Meta/UnusedNames.lean

```diff
@@ -5,7 +5,7 @@ Authors: Jannis Limperg
 -/
 
 import Lean.Meta
-import Std.Data.String
+import Std.Data.String.Basic
```

Comment:
(that said, within Std itself we should be using minimal imports, so this could go in a separate PR)